src/core.jl

```diff
@@ -1,16 +1,18 @@
 export DSSCore
 
 "Module with the low-level API for OpenDSSDirect"
-module DSSCore 
+module DSSCore
 
 if is_windows()
     if Int == Int64
         const dsslib = "$(dirname(@__FILE__))\\..\\deps\\win64\\OpenDSSDirect.DLL"
     else
         const dsslib = "$(dirname(@__FILE__))\\..\\deps\\win32\\OpenDSSDirect.DLL"
     end
-else
-    const dsslib = "$(dirname(@__FILE__))/../deps/libopendssdirect.so"
+elseif is_linux()
```

Comment:
Checking binaries in to git isn't generally a good idea. If someone is using an exotic platform usually they're on their own, but if they're able to get the library compiled, they shouldn't have to modify git-tracked files to override your default binary.